### PR TITLE
fix probs name which without temp scaling name

### PR DESCRIPTION
**Pull request by narutolhy**
## Motivation

Clarify the intent of the probs_without_temp_scaling tensor in logprob computation by assigning a more descriptive and consistent variable name. This improves readability and prevents confusion when debugging or maintaining code related to temperature scaling and RETURN_ORIGINAL_LOGPROB.

## Modifications

Renamed the temporary variable from logprobs to probs_without_temp_scaling to better reflect that the tensor represents unscaled softmax probabilities used for original logprob computation.

No change in functionality or behavior; this is a pure refactor for clarity.

Before:
`logprobs = torch.softmax(logits, dim=-1)`

After:
`probs_without_temp_scaling = torch.softmax(logits, dim=-1)`

## Accuracy Tests

This change does not affect any model outputs — logprobs are computed exactly the same way. Verified by rerunning:
'test_original_logprobs.py'

## Checklist

- [ X] Format your code according to the [Format code with pre-commit](https://docs.sglang.ai/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.ai/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.ai/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.ai/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.ai/developer_guide/contribution_guide.html#benchmark-the-speed).

### CODE REVIEW

File: python/sglang/srt/layers/sampler.py [hunk 1]
```diff
@@ -123,8 +123,8 @@ def forward(
             if return_logprob:
                 # clamp to avoid -inf
                 if RETURN_ORIGINAL_LOGPROB:
-                    logprobs = torch.log(logprobs).clamp(
-                        min=torch.finfo(logprobs.dtype).min
+                    logprobs = torch.log(probs_without_temp_scaling).clamp(
+                        min=torch.finfo(probs_without_temp_scaling.dtype).min
                     )
```
Comment (merrymercy) [inline python/sglang/srt/layers/sampler.py:128]: delete `probs_without_temp_scaling` after usage

**Comment by narutolhy** [comment]
Hi @merrymercy @zhaochenyang20 , does this pr have any other problems? Thank you

**Review by zhaochenyang20 — APPROVED** [review]
Ready to merge
